rules/windows/credential_access_dcsync_user_backdoor.toml

```diff
@@ -0,0 +1,78 @@
+[metadata]
+creation_date = "2024/07/10"
+integration = ["system", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/07/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the modification of the nTSecurityDescriptor attribute in a domain object with rights related to DCSync to a
+user/computer account. Attackers can use this backdoor to re-obtain access to hashes of any user/computer.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.security*", "logs-windows.forwarded*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential Active Directory Replication Account Backdoor"
+note = """## Setup
+
+The 'Audit Directory Service Changes' logging policy must be configured for (Success, Failure).
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Policies >
+Windows Settings >
+Security Settings >
+Advanced Audit Policies Configuration >
+Audit Policies >
+DS Access >
+Audit Directory Service Changes (Success,Failure)
+```
+"""
+references = [
+    "https://twitter.com/menasec1/status/1111556090137903104",
+    "https://www.specterops.io/assets/resources/an_ace_up_the_sleeve.pdf",
+    "https://github.com/SigmaHQ/sigma/blob/master/rules/windows/builtin/security/win_security_account_backdoor_dcsync_rights.yml"
+]
+risk_score = 47
+rule_id = "f8822053-a5d2-46db-8c96-d460b12c36ac"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Credential Access", "Data Source: Active Directory", "Use Case: Active Directory Monitoring"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.action:("Directory Service Changes" or "directory-service-object-modified") and event.code:"5136" and
+  winlog.event_data.AttributeLDAPDisplayName:"nTSecurityDescriptor" and
+  winlog.event_data.AttributeValue : (
+    (
```

Comment:
can you add comment to explain those guids like DS-Replication-Get-Changes-All extended right

Reply:
On the description?

Reply:
As KQL doesn't support comments in the query, I'll add the rights documentation pages in the references